src/index.html

```diff
@@ -2,11 +2,352 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="viewport"
+          content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+          rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+          rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__container">
+        <a href="#home" class="header__moyo-logo"></a>
+        <nav class="nav">
+          <ul class="nav__items">
+            <li class="nav__items-item">
+              <a href="#apple" class="nav__link is-active">apple</a>
+            </li>
+            <li class="nav__items-item">
+              <a href="#samsung" class="nav__link">samsung</a>
+            </li>
+            <li class="nav__items-item">
+              <a href="#smartphones" class="nav__link">smartphones</a>
+            </li>
+            <li class="nav__items-item">
+              <a href="#laptops-and-computers"
+                 class="nav__link"
+                 data-qa="nav-hover">
+                 laptops & computers
+              </a>
+            </li>
+            <li class="nav__items-item">
+              <a href="#gadgets" class="nav__link">gadgets</a>
+            </li>
+            <li class="nav__items-item">
+              <a href="#tablets" class="nav__link">tablets</a>
+            </li>
+            <li class="nav__items-item">
+              <a href="#photo" class="nav__link">photo</a>
+            </li>
+            <li class="nav__items-item">
+              <a href="#video" class="nav__link video-selector">video</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+    <main class="grid">
+      <article class="card" data-qa="card">
+        <img src="./images/imac.jpeg"
+             alt="imac-picture"
+             class="card__image">
+
+        <h2 class="card__product-name text">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <h3 class="card__product-code text">Product code: 195434</h3>
```

Comment:
```suggestion
        <h2 class="card__product-name">
          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
        </h2>

        <h3 class="card__product-code">Product code: 195434</h3>
```
There shouldn't be any additional blocks, only card and stars